packages/lumx-react/src/components/select/SelectMultiple.stories.tsx

```diff
@@ -197,7 +197,7 @@ export const chipsCustomSelectMultiple = ({ theme }: any) => {
                 onClick={onClick}
                 theme={theme}
             >
-                -> {choice}
```

Comment:
:wtf:

Reply:
A warning was triggered during the tests :rofl: